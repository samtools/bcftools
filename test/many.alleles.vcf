##fileformat=VCFv4.1
##contig=<ID=chr7>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SRR1528791
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCC,CCATCTCTCTCCCCG,CCTTCTCTCTGCCCC,CCCTCTCTCTCCCCC,CCACCTCTCTGCCCC,CCACCCCTCTGCCCA,CCACCTCCCTCCCCC,CCACCTCCCTGCCCA,CCACCTCTCGGCCCC,CCACCTCTCGGCCCA,CCACCTCTCTCCCCC,CCACCTCTCTCCCCA,CCATCCCCCTCCCCC,CCATCCCCCTGCCCC,CCATCCCCCTGCCCA,CCATCCCTCGGCCCC,CCATCCCTCGGCCCA,CCATCCCTCTCCCCC,CCATCCCTCTCCCCA,CCATCCCTCTGCCCC,CCATCTCCCGGCCCC,CCATCTCCCGGCCCA,CCATCTCCCGCCCCC,CCATCTCCCTCCCCC,CCATCTCCCTCCCCA,CCATCTCCCTGCCCC,CCATCTCTCGGCCCC,CCATCTCTCCCCCCC,CCATCTCTCGCCCCC,CCATCTCTGGGCCCA,CCATCTCTCCCCCCA,CCATCTCTCGCCCCA,CCATCTCTCTCCCCA	.	.	.	GT	2/2
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCC,CCATCTCTCTCCCCG,CCTTCTCTCTGCCCC,CCCTCTCTCTCCCCC,CCACCTCTCTGCCCC,CCACCCCTCTGCCCA,CCACCTCCCTCCCCC,CCACCTCCCTGCCCA,CCACCTCTCGGCCCC,CCACCTCTCGGCCCA,CCACCTCTCTCCCCC,CCACCTCTCTCCCCA,CCATCCCCCTCCCCC,CCATCCCCCTGCCCC,CCATCCCCCTGCCCA,CCATCCCTCGGCCCC,CCATCCCTCGGCCCA,CCATCCCTCTCCCCC,CCATCCCTCTCCCCA,CCATCCCTCTGCCCC,CCATCTCCCGGCCCC,CCATCTCCCGGCCCA,CCATCTCCCGCCCCC,CCATCTCCCTCCCCC,CCATCTCCCTCCCCA,CCATCTCCCTGCCCC,CCATCTCTCGGCCCC,CCATCTCTCCCCCCC,CCATCTCTCGCCCCC,CCATCTCTGGGCCCA,CCATCTCTCCCCCCA,CCATCTCTCGCCCCA,CCATCTCTCTCCCCA,CCTTCTCTCTGCCCC	.	.	.	GT	2/2
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCC,CCATCTCTCTCCCCG,CCTTCTCTCTGCCCC,CCCTCTCTCTCCCCC,CCACCTCTCTGCCCC,CCACCCCTCTGCCCA,CCACCTCCCTCCCCC,CCACCTCTCGGCCCC,CCACCTCTCGGCCCA,CCACCTCTCTCCCCC,CCACCTCTCTCCCCA,CCATCCCCCTCCCCC,CCATCCCCCTGCCCC,CCATCCCCCTGCCCA,CCATCCCTCGGCCCC,CCATCCCTCGGCCCA,CCATCCCTCTCCCCC,CCATCCCTCTCCCCA,CCATCCCTCTGCCCC,CCATCTCCCGGCCCC,CCATCTCCCGGCCCA,CCATCTCCCGCCCCC,CCATCTCCCTCCCCC,CCATCTCCCTCCCCA,CCATCTCCCTGCCCC,CCATCTCTCGGCCCC,CCATCTCTCCCCCCC,CCATCTCTCGCCCCC,CCATCTCTGGGCCCA,CCATCTCTCCCCCCA,CCATCTCTCGCCCCA,CCATCTCTCTCCCCA,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC,CCTTCTCTCTGCCCC	.	.	.	GT	2/2
