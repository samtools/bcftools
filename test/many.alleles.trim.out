##fileformat=VCFv4.1
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=chr7>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SRR1528791
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCG	.	.	AC=2;AN=2	GT	1/1
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCG	.	.	AC=2;AN=2	GT	1/1
chr7	142459798	.	CCATCTCTCTGCCCA	CCATCTCTCTCCCCG	.	.	AC=2;AN=2	GT	1/1
